test/cctest/wasm/test-run-wasm.cc

```diff
@@ -765,6 +765,171 @@ TEST(Run_WasmInt64Binops) {
   TestInt64Cmp(kExprI64LtU, 1, 0, -6);
   TestInt64Cmp(kExprI64LeU, 1, 98978, 0xF0000000);
 }
+
+
+void TestInt64ToInt32Unop(WasmOpcode opcode, int32_t expected, int64_t a) {
+  {
+    WasmRunner<int32_t> r;
+    // return op K
+    BUILD(r, WASM_UNOP(opcode, WASM_I64(a)));
+    CHECK_EQ(expected, r.Call());
+  }
+  {
+    WasmRunner<int32_t> r(kMachInt64);
+    // return op a
+    BUILD(r, WASM_UNOP(opcode, WASM_GET_LOCAL(0)));
+    CHECK_EQ(expected, r.Call(a));
+  }
+}
+
+
+TEST(Run_WasmInt64Clz) {
+  TestInt64ToInt32Unop(kExprI64Clz, 0, 0x8000100000000000);
```

Comment:
These build a new graph, do a fresh compile, and run each time. Can you refactor this code to build and compile once?